chimera_ctioenviroment/instruments/lcogtweather.py

```diff
@@ -90,12 +87,13 @@ def _update(self, value):
         """
         Updates with the LCOGT results
         """
-        if all([v in value for v in
-                ['Humidity', 'Pressure', 'Temperature', 'Transparency', 'Dew Point', 'Wind', 'OK to open']]):
+        if all([v in value.keys() for v in
+                ['humidity', 'pressure', 'temperature', 'sky_transparency', 'dew_point',
+                 'wind_speed', 'wind_direction']]):
             if not value.has_key('Interlock Reason'):
                 value['Interlock Reason'] = ''
             self._results = value
-            self._results['utctime'] = datetime.datetime.strptime(value['utctime'], '%Y-%m-%d %H:%M UTC')
+            self._results['utctime'] = datetime.datetime.strptime(value['temperature']['TimeStamp'].replace('/', '-'), '%Y-%m-%d %H:%M:%S')
```

Comment:
can't you change `'%Y-%m-%d %H:%M:%S'` to `'%Y/%m/%d %H:%M:%S'` to remove the `replace()`?

Reply:
Done